### PR TITLE
Area_01 finally completed!

### PR DESCRIPTION
**Pull request by azazellus**
- Puzzles and timing have been tuned;
- The player now moves correctly for 60fps;
- The quality settings are now set to "don't sync", so now the 60fps limit works in all the quality presets;
- I modified a noclip script to disable a useless hint;
- I also modified (badly) the menu and the gui scenes, but let's keep those from the develop. Just remember to add the new menu background (OneDrive).

### CODE REVIEW

**Review by bottolo — DISMISSED** [review]
Works, and finally that hint container works properly.

**Comment by TizioMaurizio** [comment]
Good, honestly i would increase the time of all puzzles by 50 percent because there are certain puzzles where the time is so strict that you can't stop and in the demo especially this can cause frustration, also the last 2 puzzles are already difficult enough without time, but +50% should be enough anyway

**Comment by TizioMaurizio** [comment]
also i would add message when the game is finished by both complimenting the player, asking him for a feedback and telling him that he can exit the game

**Comment by TizioMaurizio** [comment]
and last thing the last puzzle is too tedious, maybe we can just remove it since there is no way someone can finish the demo with that, the one right before is already quite time consuming

**Review by TizioMaurizio — DISMISSED** [review]
tweak area 01 new (and maybe rename it)

**Comment by bottolo** [comment]
@TizioMaurizio stfu we need to deliver asap

File: Assets/Code/Scripts/LevelManagement/GameManager.cs [hunk 1]
```diff
@@ -35,6 +35,7 @@ private void Awake()
     private void Start()
     {
         Application.targetFrameRate = 60;
+        QualitySettings.vSyncCount = 1;
```
Comment (stefanofossati) [inline Assets/Code/Scripts/LevelManagement/GameManager.cs:38]: delete the line

Comment (bottolo) [reply Assets/Code/Scripts/LevelManagement/GameManager.cs:38]: no

**Comment by TizioMaurizio** [comment]
> @TizioMaurizio stfu we need to deliver asap

👺

**Review by bottolo — APPROVED** [review]
Works. We'll see about the vsync thing. Also, GORGEOUS MENU!

**Comment by bottolo** [comment]
Menu stuff added here, at last.

**Pull request merged**